### DISCUSSION TITLE
Patch Panel, Racks etc. should be separated from Managed devices.

### DISCUSSION THREAD

**Discussion by digital-fortress** [netbox-community/netbox#15132] [Ideas]
I think it would be helpful if in NetBox, we separate things like patch panels, racks, and other infrastructure components from actual managed devices. When we're checking out the devices list on a site, it should only include things like managed switches and routers, not the racks and patch panels. It makes things clearer and easier to manage.

**Comment by PaulR282** [comment]
Have a look at device roles. This is exactly what they are for.

**Reply by ElonLP** [reply]
But then it is also in the main device list. Is there any other solution, without separately set an filter?

**Reply by digital-fortress** [reply]
Right, for example , the way modules are categorized. They are not shown in the main device list.

**Reply by ElonLP** [reply]
Yes, like modules but positionable like Devices. It would actually be nice if patch panels and blind panels had their own rack accessory category.

**Reply by Supermathie** [reply]
Is there a way to do exclusion filters?

e.g. instead of `/dcim/devices/?role_id=42`, `/dcim/devices/?role_id__not=42`

**Reply by alehaa** [reply]
Try `role_id__n=42`.

**Comment by digital-fortress** [comment]
Will this be implemented ?

**Reply by alehaa** [reply]
You need to file a feature request under issues for an answer to this question.
